docs/contributing.md

```diff
@@ -436,13 +433,15 @@ Installing the pre-commit hooks to git
 
 1. (Extra info) More commands to test run the hooks
 
-    ```bash
-    pre-commit run --all-files --hook-stage push
-    pre-commit run --all-files --hook-stage commit
-    pre-commit run test --all-files --hook-stage push
-    ```
+   ```bash
+   pre-commit run --all-files --hook-stage push
+   pre-commit run --all-files --hook-stage commit
+   pre-commit run test --all-files --hook-stage push
+   ```
+
+# Appendix
 
-### Convenience scripts for sanity checks before committing code (assumes bash env)
+# A. Convenience scripts for sanity checks before committing code (assumes bash env)
```

Comment:
`# A. Convenience Scripts for Sanity Checks Before Committing Code (assumes bash env)`

Reply:
Done.  Also removed (assumes bash env) and added sentence to beginning "These scripts assume you are using bash."